src/index.html

```diff
@@ -22,6 +22,349 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="header">
+        <a
+          class="logo"
+          href="index.html"
+        >
+          <img
+            src="images/logo.png"
+            alt="moyo-logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#section-1"
+                class="nav__link is-active"
+              >
+                APPLE
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-2"
+                class="nav__link"
+              >
+                SAMSUNG
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-3"
+                class="nav__link"
+              >
+                SMARTPHONES
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-4"
+                data-qa="nav-hover"
+                class="nav__link laptops"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-5"
+                class="nav__link"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-6"
+                class="nav__link"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-7"
+                class="nav__link"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-8"
+                class="nav__link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="products">
+      <section
+        class="products__card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="IMAC image"
+          class="products__card_image"
```

Comment:
The class `products__card_image` should follow the BEM naming convention. Consider renaming it to `products__card-image` to maintain consistency.